DoSomethingArchive/legacy-website
# Update win module background image

#### What's this PR do?
- Removes the win module background image config field on the backend. 
- Adds and sets the win module background pattern as an SVG in the theme.
#### Where should the reviewer start?

`dosomething_campaign.admin.inc` is where the config field is being removed and `dosomething_campaign.install` is where the variable gets deleted permanently.

`_win-module.scss` is where we set the background image in the theme.
#### How should this be manually tested?

Check out a campaign in the win phase and the image should be sharper because we are using an SVG instead of rasterized images. 

In the backend, if you go to `configuration -> dosomething_config -> dosomething_campaign` you should no longer see a field for setting the background image in the win modules settings.
#### What are the relevant tickets?

Fixes #5011 

@DoSomething/front-end for frontend stuff
@angaither for the bender-y removing the config field stuff.
